app/inputs/select_with_modal_help_input.rb

```diff
@@ -31,6 +31,13 @@ def build_field(value, _index)
       @rendered_first_element = true
 
       html_options.merge!(options.slice(:include_blank))
-      template.select_tag(attribute_name, template.options_for_select(select_options, value), html_options)
+      render_options = select_options
```

Comment:
D'oh! Never mind. The variable is left over from a previous iteration of the fix where the forced option was added to the select at render time, not at runtime.